### ISSUE TITLE
Summary page (/analytics/dataset/top) not working on CKAN 2.9

### ISSUE THREAD

**Issue by Sean-Stilwell** [ckan/ckanext-googleanalytics#65]
I've got the extension working on version 2.9 and am able to view the stats on Google Analytics & use `ckan -c /etc/ckan/default/ckan.ini googleanalytics load mycredentials.json` to update the database.

Unfortunately, I'm running into issues with this part:

> 7. Look at some stats within CKAN

> Once your GA account has gathered some data, you can see some basic information about the most popular packages at: http://mydomain.com/analytics/dataset/top

I've tried visiting localhost:5000/analytics/dataset/top but that's just giving a 404 error. Not sure if this page is working on CKAN 2.9 or if I've messed up somewhere along the way.

**Comment by smotornyuk** [comment]
No, you are absolutely right, this page does not exist. At the moment ckanext-googleanalytics does not provide any page(or another way) to view statistics. And README just must be updated.

[This extension](https://github.com/DataShades/ckanext-ga-report/tree/py3) adds the visual part somewhere under `/site-usage` path. But it's not actively supported, so don't expect much from it
